src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -3043,6 +3049,14 @@ public override BoundNode VisitReadOnlySpanFromArray(BoundReadOnlySpanFromArray
             }
         }
 
+        /// <summary>
+        /// This visitor represents just the non-assignment part of the null coalescing assignment
+        /// operator (when the left operand is non-null).
+        /// </summary>
+        protected virtual void VisitUnassignedLeftOfNullCoalescingAssignment(BoundNullCoalescingAssignmentOperator node)
```

Comment:
>VisitUnassignedLeftOfNullCoalescingAssignment [](start = 31, length = 45)

The name of this method feels very confusing to me. It is not meant to visit anything, especially not meant to visit Left. AdjustTheStateForNonNullCase?